3news.co.nz.js

```diff
@@ -1,282 +0,0 @@
-{
-	"translatorID": "a9f7b277-e134-4d1d-ada6-8f7942be71a6",
```

Comment:
No, better to use the same ID, since this should replace the old one. (Also otherwise the old one would have to be marked as deleted separately.)